src/utils/Obs_NumberHelper.cpp

```diff
@@ -31,8 +31,15 @@ uint64_t Utils::Obs::NumberHelper::GetOutputDuration(obs_output_t *output)
 	video_t *video = obs_output_video(output);
 	uint64_t frameTimeNs = video_output_get_frame_time(video);
 	int totalFrames = obs_output_get_total_frames(output);
-
-	return util_mul_div64(totalFrames, frameTimeNs, 1000000ULL);
+	uint64_t divisor = 0ULL;
```

Comment:
Nitpick: Add a newline after `totalFrames` and before the return statement.

Also, a comment along the lines of "This is a hack, someone please improve this in the future." is probably fair to have in there.